src/app/shared/abstract/service/editors/preview.service.ts

```diff
@@ -0,0 +1,56 @@
+import { Injectable } from '@angular/core';
+import { ITEM_CONSTANTS } from 'app/features/item/item-template/item-constants';
+import { RACE } from 'app/features/item/item-template/item-preview';
+
+@Injectable()
+export class PreviewService {
```

Comment:
should be renamed to `PreviewHelperService` and provided either in a common module or in root